Makefile

```diff
@@ -1,7 +1,9 @@
 -include ../../config.mk
 -include ../../tools/kconfig/.config
 
-CC:=$(CROSS_COMPILE)gcc
+OS:=$(shell uname)
+
+CC:=$(CROSS_COMPILE)clang
```

Comment:
You could run `make CC=clang`, that overrides the default that's set in the file